tests/conftest.py

```diff
@@ -47,7 +64,7 @@ def ansible_zos_module(request, z_python_interpreter):
         host.vars["ansible_python_interpreter"] = interpreter
         host.vars["ansible_connection"] = "zos_ssh"
     yield adhoc
-    adhoc.all.command(cmd="opercmd '$PJ(*)'")
+    clean_logs(adhoc)
```

Comment:
This is a question, not necessarily a criticism (just curious), if `clean_logs` is being executed after every test case, would that be time consuming to test suites that have a large number of test cases? I understand that it is probably necessary to cleanup WTOR buffer, but still curious.